text/0040-pagination-convention.md

```diff
@@ -0,0 +1,47 @@
+- Title: Pagination Convention
+- Start Date: 2021-05-10
+- Specification PR: [#40](https://github.com/meilisearch/specifications/pull/40)
+- MeiliSearch Tracking-Issues:
+
+# Pagination Convention
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Pagination allows the content to be viewed in the form of a sequence in order to facilitate the browsing of its content by not overloading the number of elements to be viewed for the user. Pagination also has the advantage of easily providing the total number of items that remain to be traversed by not overloading the network with all the content that can be loaded.
+
+### II. Motivation
+
+This specification will be used as a convention to implement simple and standard paginations to be set up on API routes such as the list of updates for example. Since the pagination of search results is intrinsically linked to technical implementations of the search engine, this specific case is detailed in its own specification.
+
+### III. Additional Materials
+N/A
+
+### IV. Explanation
+
+##### Requirements
```

Comment:
For some routes, like `/updates`, I would rather have the last update sent at the top of the list rather than at the end. But maybe I'm the only one.

Reply:
Yes, this example is to illustrate the pagination convention. A future specification on `/updates` will allow to define this specific behavior for the update API route. It could also be an asc/dec filter for example.

Reply:
Awesome! Thanks :)